test/unit/cookies.test.js

```diff
@@ -122,6 +123,138 @@ describe('cookie configuration', function () {
   })
 })
 
+describe('reduce session charge count', () => {
+  let initialEnvironmentVariables
+  before(() => {
+    initialEnvironmentVariables = Object.assign({}, process.env)
+  })
+
+  afterEach(() => {
+    for (const envVar in process.env) {
+      process.env[envVar] = initialEnvironmentVariables[envVar]
```

Comment:
This doesn't quite reset them to their previous state, I'd suggest checking `initialEnvironmentVariables.hasOwnProperty(envVar)` and if that's false running a `delete proces.env[envVar]`

Reply:
interesting, this was copied from another test so we'll need to revisit other places this is used